com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 ### Fixed
 
+- Fixed issue where server listen address was being assigned 127.0.0.1 when it was null.
```

Comment:
Don't think we need this here, since this wasn't broken in any public releases.

Reply:
Awe... I will revert but then I need to make some other change or the PR will fail... being only 1 file will have changed.

Reply:
I don't think there's a rule against 1-file change sets...?

Reply:
Or maybe it just requires more than 1 commit... we will see... 👀

Reply:
it only requires more than 1 commit if the commit description doesn't follow the `fix:` naming convention.